cinema/serializers.py

```diff
@@ -51,7 +60,24 @@
 
     class Meta:
         model = Movie
-        fields = ("id", "title", "description", "duration", "genres", "actors")
+        fields = (
+            "id",
+            "title",
+            "description",
+            "duration",
+            "genres",
+            "actors",
+            "image"
```

Comment:
Code Efficiency Issue: The same issue is here in the `MovieDetailSerializer`. You should use `serializers.ImageField` for the `image` field.